### PR TITLE
PRM Automated Test - LPS-169960 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LPS-169960

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFListing.path [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">PRMMDFListing</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>LISTING_TABLE_NAME</td>
+	<td>//*[@route='mdf-request-list']//thead//th/p[contains(text(),'Name')]//ancestor::table//tbody//th[3][normalize-space(text())='${key_campaignName}']</td>
+	<td></td>
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFListing.path:16]: Is there a way to create this path without considering th[3] ?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+</tr>
+<tr>
+	<td>ADD_BUTTON</td>
+	<td>//*[contains(@class,'lexicon-icon-plus')]//ancestor::button[contains(text(),'${key_buttonText}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>LISTING_TABLE_NAME</td>
+	<td>//*[@route='mdf-request-list']//thead//th/p[contains(text(),'Name')]//ancestor::table//tbody//th[6][contains(text(),'Campaign Name')]</td>
+	<td></td>
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path:41]: Is there a way to consider this path without considering th[6]?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+				key_fieldName = "activities[${activityIndex}].leadFollowUpStrategies",
+				key_optionValue = "${leadFollowUp}",
+				locator1 = "PRMMDFRequest#CHECKBOX");
+		}
+
+		Type(
+			key_fieldName = "activities[${activityIndex}].detailsLeadFollowUp",
+			locator1 = "PRMMDFRequest#TEXT_FIELD",
+			value1 = "${followUpStrategy}");
+	}
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro:189]: This macro looks so big... is there a way to split it? Let me know if there isn't as well, I trust your judgement.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFListing.path (hunk 1 above)
Comment (AfonsoHSantos) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFListing.path:16]: I'll work on this

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro (hunk 3 above)
Comment (AfonsoHSantos) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro:189]: I think that it's not possible. This section from PRM is a form with a lot of fields.

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**